<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Malaviyan" failures="1" tests="5" name="Test" time="0.167" errors="0" timestamp="2025-09-14T12:39:36 IST">
  <testcase classname="com.automation.day1.TesterDemo" name="SIgnIn" time="0.028"/>
  <testcase classname="com.automation.day1.TesterDemo" name="SignIn" time="0.003"/>
  <testcase classname="com.automation.day1.PreAssume" name="test3" time="0.004"/>
  <testcase classname="com.test.day2.LoginTest" name="getTitle" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at com.test.day2.LoginTest.getTitle(LoginTest.java:17)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- getTitle -->
  <testcase classname="com.automation.day1.Demo" name="prinHelloTest" time="0.002"/>
</testsuite> <!-- Test -->
